api/src/test/java/org/openmrs/module/ugandaemrpoc/api/dao/UgandaEMRPocDaoTest.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.ugandaemrpoc.api.dao;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+/**
+ * It is an integration test (extends BaseModuleContextSensitiveTest), which verifies DAO methods
+ * against the in-memory H2 database. The database is initially loaded with data from
+ * standardTestDataset.xml in openmrs-api. All test methods are executed in transactions, which are
+ * rolled back by the end of each test method.
+ */
+public class UgandaEMRPocDaoTest extends BaseModuleContextSensitiveTest {
```

Comment:
@slubwama Remove this class

Reply:
@ssmusoke These are default classes

Reply:
@slubwama As a principle: Do not keep any code not being used - we do not use the DAO and probably will not.